src/figuresClasses.ts

```diff
@@ -1,11 +1,91 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
```

Comment:
Create separate type  `type Shape = 'triangle' | 'circle' | 'rectangle'` and use it here